setup.py

```diff
@@ -25,6 +25,8 @@
     setup_requires=[
         'mock>=1.0.1',
         'testfixtures>=4.1.2',
+        'troposphere>=1.0.0',
```

Comment:
Given troposphere is already in the install_requires list we shouldn't need to add it here. What does this fix or what breaks without it?